common/src/models/response/errorResponse.ts

```diff
@@ -20,6 +23,8 @@ export class ErrorResponse extends BaseResponse {
         if (errorModel) {
             this.message = this.getResponseProperty('Message', errorModel);
             this.validationErrors = this.getResponseProperty('ValidationErrors', errorModel);
+            this.captchaSiteKey = this.validationErrors?.HCaptcha_SiteKey?.[0];
```

Comment:
[Question] I know we abstracted the `HCaptcha_SiteKey` server-side, I wonder if there's a way to do the same here or not, or if the fact we're using HCaptcha unavoidable in the front-end anyway?

Reply:
I thought about that but ultimately decided that it'd be premature generalization.

If we really think we might use another provider I think the most prudent thing to do right now is set the `HCaptcha_SiteKey` key name in another property so we can sync it with what the server is requesting. Do you think there's value in that?

Reply:
Not at this time, no, thanks for the follow up!